src/worker/worker/collectors/simple_web_collector.py

```diff
@@ -23,35 +22,33 @@ def __init__(self):
 
     def parse_source(self, source):
         super().parse_source(source)
-        self.web_url = source["parameters"].get("WEB_URL", None)
-        if not self.web_url:
+        self.collector_url = source["parameters"].get("WEB_URL", None)
```

Comment:
collector_url is also confusing.

Streamlining it between the different collectors could make sense, but it would require a better name. In this case I would rather stay with web_url and feed_url

Reply:
we need to come up with a common variable because we use it in `get_urls()`, this solves the bug that the RSS collector couldn't Digest Split.

Reply:
just pass the url as a parameter to `get_urls()`